test/index.js

```diff
@@ -26,7 +26,7 @@ test('Compare', function(t) {
 
   var compare = new mapboxgl.Compare(a, b, container);
 
-  t.notOk(!!a.getContainer().style.clip, 'Map A is not clipped');
+  t.ok(!!a.getContainer().style.clip, 'Map A is clipped');
```

Comment:
Why did the result of this test change?

Reply:
Someone changed the behavior and forgot to update the test, it was failing as is, figured I'd fix it.